custom_components/zwave_mqtt/entity.py

```diff
@@ -126,6 +126,10 @@ def _check_entity_ready(self):
         if component in PLATFORMS:
             async_dispatcher_send(self._hass, f"zwave_new_{component}", self)
 
+    @property
+    def entity(self):
```

Comment:
Expose an id instead. The entity should know about this id so the entity can listen to signals for this id.

Reply:
Yeah, that's exactly what I was missing. The id is a better/safer choice indeed.